sectionTwoCode/class15.txt

```diff
@@ -0,0 +1,55 @@
+//need add super() at extends class
+class Human {
+	constructer(){
+		this.gender = 'male';
+	}
+	
+	printGender(){
+		console.log(this.gender);
+	}
+}
+
+class Person extends Human {
+	constructer(){
+		super();
+		this.name = 'Max';
+	}
+	
+	printMyName() {
+		console.log(this.name);
+	} 
+}
+
+const person = new Person();
+person.printMyName();
+person.printGender();
+
+
+//add gender at extends class will update the value at human class
+
+class Human {
+	constructer(){
+		this.gender = 'male';
+	}
+	
+	printGender(){
+		console.log(this.gender);
+	}
+}
+
+class Person extends Human {
```

Comment:
Try to review your course and get to know why use `extends` and how?

Reply:
Extending a Component allows us to pass props to a user defined class when a constructor is not present like it is in the App class